mlflow/models/evaluation/base.py

```diff
@@ -758,7 +934,9 @@ def evaluate(
         - The evaluation dataset label values must be numeric or boolean, all feature values
           must be numeric, and each feature column must only contain scalar values.
 
-    :param model: A pyfunc model instance, or a URI referring to such a model.
+    :param model: A pyfunc model instance, or a URI referring to such a model, as
+                  the candidate_model to be compared with baseline_model
+                  (specified by the `baseline_model` param) for model validation.
```

Comment:
I think that the documentation of ``baseline_model`` is sufficient to explain the role of candidate / baseline models.

```suggestion
    :param model: A pyfunc model instance, or a URI referring to such a model.
```